package/gluon-mesh-vpn-tunneldigger/files/lib/gluon/upgrade/400-mesh-vpn-tunneldigger

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site_config'
+local users = require 'gluon.users'
+local util = require 'gluon.util'
+
+local uci = require('luci.model.uci').cursor()
+local lutil = require 'luci.util'
+
+-- Group for iptables rule
+users.add_group('gluon-tunneldigger', 900)
+
+local enabled = uci:get_first('tunneldigger', 'broker', 'enabled')
+if (tonumber(enabled) == 0) then
+  enabled = site.tunneldigger_mesh_vpn.enabled and 1 or 0
+end
+
+-- Delete old broker config section       
+uci:delete_all('tunneldigger', 'broker')
+
+section = uci:add('tunneldigger', 'broker')
+
+uci:section('tunneldigger', 'broker', section,
```

Comment:
It would be nicer to use a static section name like `mesh_vpn` for the broker instead of referring to it using `@broker[0]`, so this can't collide with other tunneldigger configurations (that is, if the tunneldigger init script can handle multiple instances, which I didn't check)